src/main/java/funbox/util/Parser.java

```diff
@@ -0,0 +1,151 @@
+package funbox.util;
+
+import java.time.LocalDate;
+
+import funbox.command.ByeCommand;
+import funbox.command.Command;
+import funbox.command.DeadlineCommand;
+import funbox.command.DeleteCommand;
+import funbox.command.EventCommand;
+import funbox.command.FilterCommand;
+import funbox.command.FindCommand;
+import funbox.command.ListCommand;
+import funbox.command.MarkCommand;
+import funbox.command.ToDoCommand;
+import funbox.command.UnmarkCommand;
+import funbox.exception.FunBoxExceptions;
+
+/**
+ * Deal with parsing and making sense of the user commands.
+ */
+public class Parser {
+
+    public final String unknownCommand = "Sorry! You have used the command wrongly!";
```

Comment:
coding style - I think it might be better to use capital letters for the final string?